dev/README.md

```diff
@@ -31,3 +34,38 @@ The script currently has the following limitations:
 - It only works with Excel spreadsheets as input.
 - It does not generate default values for rules. Default values must be added manually if they are not present in the input spreadsheet.
 - Rules rego implementation is required before running the script. The script will fail if the rego implementation is not present.
+
+### Generate Rule Templates
+
+`generate_rule_templates.py` generate the rule templates that will show in our Kibana plug-in (csp-rules).
+
+**Usage:**
+
+From the root dir you can run the following example to generate selected benchmark rules templates
+
+```shell
+poetry run python dev/generate_rule_templates.py --benchmark <benchmark_id> --rules <selected rules>
+```
+
+**Example 1** - Generate all rules templates from all benchmarks:
+
+```shell
+poetry run python dev/generate_rule_templates.py
+```
+
+**Example 2** - Generate two specific rules templates from CIS AWS:
+
+```shell
+poetry run python dev/generate_rule_templates.py --benchmark cis_aws --rules "1.8" "1.9"
+```
+
+**Example 3** - Generate two specific rules templates from CIS AWS and save them in a different directory:
+
+```shell
```

Comment:
Maybe it worth mentioning that it will be created under the `dev` directory - definitely not a must